examples/winit_multithread.rs

```diff
@@ -19,16 +19,16 @@ mod ex {
 
     fn render_thread(
         window: Arc<Window>,
-        surface: Arc<Mutex<Surface>>,
-        do_render: mpsc::Receiver<()>,
+        do_render: mpsc::Receiver<Arc<Mutex<Surface>>>,
         done: mpsc::Sender<()>,
     ) {
         loop {
             println!("waiting for render...");
-            if do_render.recv().is_err() {
+            let Ok(surface) = do_render.recv() else {
+                println!("main thread destroyed");
                 // Main thread is dead.
```

Comment:
I don't think the comment is needed if we print a line saying the same thing as well that is very human readable.